Fst2List.cpp

```diff
@@ -76,7 +80,8 @@ const char* usage_Fst2List =
         "-v, --verbose: prints information during the process\r\n"
         "-r (s|l|x) <L[,R]>: enclose loops in L and R strings as in (c0|...|cn) by Lc0|..|cnR : default null\r\n"
         "-V, --only_verify_arguments: only verify arguments syntax and exit\r\n"
-        "-h, --help: display this help and exit";
+        "-h, --help: display this help and exit\r\n"
+        "-b <file>, generate dictionary file according the given configuration file,";
```

Comment:
`-V, ...` and `-h, ...` should be the last lines when the help is displayed.

Reply:
done